MarketPlace.Backend/MarketPlace/Microservices/Order Service/Services.Order.WebAPI/Controllers/CartController.cs

```diff
@@ -29,31 +29,7 @@ public async Task<ActionResult<Cart>> GetCart(Guid id)
             return Ok(cart);    
         }
 
-        [HttpPost]
-        public async Task<ActionResult<Guid>> CreateCart(CancellationToken cancellationToken)
-        {
-            var command = new CreateCartCommand()
-            {
-                UserId = UserId
-            };
-
-            var resultId = await Mediator.Send(command, cancellationToken);
-            return Ok(resultId);
-        }
-
-        [HttpDelete("{id}")]
-        public async Task<IActionResult> DeleteCart(Guid id, CancellationToken cancellationToken)
-        {
-            var command = new DeleteCartCommand()
-            {
-                Id = id
-            };
-
-            await Mediator.Send(command, cancellationToken);
-            return Ok();
-        }
-
-        [HttpPut("AddPoint/{CartId}/{ProductId}")]
+        [HttpPost("{CartId}/Point/{ProductId}")]
```

Comment:
```suggestion
        [HttpPost("carts/{cartId}/products/{productId}")]
```
Please read the article at this link https://restfulapi.net/resource-naming/